src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -2155,13 +2105,13 @@ public override string GetName(SyntaxNode declaration)
                 case SyntaxKind.MethodDeclaration:
                     return ((MethodDeclarationSyntax)declaration).Identifier.ValueText;
                 case SyntaxKind.FieldDeclaration:
-                    return GetName(((FieldDeclarationSyntax)declaration).Declaration);
+                    return this.GetName(((FieldDeclarationSyntax)declaration).Declaration);
```

Comment:
I would prefer we just not touch 'this' (either to remove or add).  I know when i write code i pick the style for 'this' intentionally to make it readable from my perspective.  

Also i don't know how that style got picked for our project.  It was never something that i was roped into a discussion on.  I'm fine with it being set to 'false' in that any codegen features won't automatically add 'this'.  But it's not a hard rule that code should not use 'this' (or that code *should* use 'this').